images/origin/Dockerfile

```diff
@@ -2,6 +2,10 @@
 # This is the official OpenShift Origin image. It has as its entrypoint the OpenShift
 # all-in-one binary.
 #
+# While this image can be used for a simple node it does not support OVS based
+# SDN or storage plugins required for EBS, GCE, Gluster, Ceph, or iSCSI volume
```

Comment:
I'm not sure if this is true for Gluster. That plugin should not be doing format/mkfs operations, right? So it should 'work' with the base image...